### PR TITLE
Update EngDiff UI manual testing instructions

### LINKED ISSUE #35598: Update testing docs for EngDiff UI
- Typo in test 3 - custom spectra should be 1200-2400 (not 1400)
- Typo test 4 - workspace suffix is `_logs_Fitting`, also need to move explanation of `run_info` earlier as use this table for checking UI table
- Test 4 - if runs are loaded then re-loading with Add to plot won't work (because already loaded)
- Test 5 add step to delete a `_bgsub` workspace from ADS (row will not be deleted but bgsub box will be unchecked)
- Test 6 Suffix is `_bgsub` not `_bg`
- Test 7 step 6 - type suffix should be `_dSpacing`
- Test 8 - don't need to re calibrate and focus - we already have the focussed nexus for these runs
- Test 10 - move to earlier test

### PR DESCRIPTION
**Pull request by RichardWaiteSTFC**
**Description of work**
 Update testing instructions from feedback in v6.7 (no comments from v6.8 #36051)

- Typo in test 3 - custom spectra should be 1200-2400 (not 1400)
- Typo test 4 - workspace suffix is _logs_Fitting, also need to move explanation of run_info earlier as use this table for checking UI table
- Test 4 - if runs are loaded then re-loading with Add to plot won't work (because already loaded)
- Test 5 add step to delete a _bgsub workspace from ADS (row will not be deleted but bgsub box will be unchecked)
- Test 6 Suffix is _bgsub not _bg
- Test 7 step 6 - type suffix should be _dSpacing
- Test 8 - don't need to re calibrate and focus - we already have the focussed nexus for these runs
- Test 10 - move to earlier test

**To test:**

Code review and check builds pass

Fixes #35598 

---

#### Reviewer ####

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

#### Gatekeeper ####

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.

### CODE REVIEW

**Comment by thomashampson** [comment]
let's keep this going into `main` rather than `release-next`. Dev docs are always published from `main` anyway.

File: dev-docs/source/Testing/EngineeringDiffraction/EngineeringDiffractionTestGuide.rst [hunk 1]
```diff
@@ -165,18 +171,20 @@ This tests the removal of focused runs from the fitting tab.
...
-Test 6
+6. Delete a ``_bgsub`` workspace in the ADS, the corresponding row will not be deleted, but the bgsub checkbox will be unchecked.
+
+Test 7
 ^^^^^^
 
 This tests that the background subtraction works.
 
-1. Load in a run - the `Subtract BG` box should be checked in the UI table by default. This should generate a workspace with suffix `_bg` and the data should look like the background is flat and roughly zero on the plot using the default parameters (other columns in the UI table).
+1. Load in a run - the `Subtract BG` box should be checked in the UI table by default. This should generate a workspace with suffix `_bgsug` and the data should look like the background is flat and roughly zero on the plot using the default parameters (other columns in the UI table).
```
Comment (warunawickramasingha) [inline dev-docs/source/Testing/EngineeringDiffraction/EngineeringDiffractionTestGuide.rst:181]: There is a typo, it should be _bgsu**b**

File: dev-docs/source/Testing/EngineeringDiffraction/EngineeringDiffractionTestGuide.rst [hunk 2]
```diff
@@ -137,14 +137,14 @@ This test covers the loading and plotting focused data in the fitting tab.
 
 1. In the fitting tab, load multiple of these newly focused TOF .nxs files in the `Load Focused Data` section. The path to the focused files should be auto populated
 
-2. Click the `Load` button. A row should be added to the UI table for each focused run.There should be a grouped workspace with the suffix `_logs` in the ADS with tables corresponding to each log value specified in the settings (to open the settings use the cog in the bottom left corner of the UI). Each row in these tables should correspond to the equivalent row in the UI table. There should be an additional table called `run_info` that provides some of the metadata for each run.
+2. Click the `Load` button. A row should be added to the UI table for each focused run.There should be a grouped workspace with the suffix `_logs_Fitting` in the ADS with tables corresponding to each log value specified in the settings (to open the settings use the cog in the bottom left corner of the UI). Each row in these tables should correspond to the equivalent row in the UI table. There should be an additional table called `run_info` that provides some of the metadata for each run.
```
Comment (warunawickramasingha) [inline dev-docs/source/Testing/EngineeringDiffraction/EngineeringDiffractionTestGuide.rst:140]: I think this description of 'run_info' is needed be moved earlier

File: dev-docs/source/Testing/EngineeringDiffraction/EngineeringDiffractionTestGuide.rst
Comment (warunawickramasingha) [inline dev-docs/source/Testing/EngineeringDiffraction/EngineeringDiffractionTestGuide.rst:1]: There is a discrepancy in the files generated at Test 1 - Focus - Step 6. The test specify as 

> `ENGINX_305761_307521_all_banks.gss` and `ENGINX_305761_307521_all_banks.abc` (i.e. two ASCII files per run - each file contains all the spectra for a focused run).

Whereas the real files generated are like 

> ENGINX_305761_307521_all_banks_dSpacing.gss, ENGINX_305761_307521_bank_2_dSpacing.gss,  ENGINX_305761_307521_all_banks_TOF.gss and ENGINX_305761_307521_bank_2_TOF.gss

> ENGINX_305761_307521_all_banks_dSpacing.abc, ENGINX_305761_307521_all_banks_TOF.abc, ENGINX_305761_307521_bank_2_dSpacing.abc, and ENGINX_305761_307521_bank_2_TOF.abc

File: dev-docs/source/Testing/EngineeringDiffraction/EngineeringDiffractionTestGuide.rst [hunk 3]
```diff
@@ -121,7 +121,7 @@ This test covers the Cropping functionality in the Calibration tab.
 
 7. Change the RB number to `Custom`.
 
-8. Set the ``Region Of Interest`` to ``Crop to Spectra`` and using ``Custom Spectra`` ``1200-1400`` (these spectrum numbers correspond to the South Bank).
+8. Set the ``Region Of Interest`` to ``Crop to Spectra`` and using ``Custom Spectra`` ``1200-2400`` (these spectrum numbers correspond to the South Bank).
    Please note that some custom spectra values may cause the algorithms to fail. Click ``Calibrate`` and a similar plot to before should appear but with only 2 subplots.
 
 9. Set the ``Region of Interest`` to ``Texture (20 spec)`` - there should be 20 spectra per run (5 tiled plot windows, 4 spectra per window).
```
Comment (warunawickramasingha) [inline dev-docs/source/Testing/EngineeringDiffraction/EngineeringDiffractionTestGuide.rst:127]: 9. Set the ``Region of Interest`` to ``Texture (20 spec)`` **and  Click 'Calibrate'**?

**Review by warunawickramasingha — CHANGES_REQUESTED** [review]
As I have commented above, there is a typo and a discrepancy in the files generated at Test 1 - Focus - Step 6.

**Comment by warunawickramasingha** [comment]
> As I have commented above, there is a typo and a discrepancy in the files generated at Test 1 - Focus - Step 6.

In addition, I noticed a few places where the screenshots of the Engdiff interface looks outdated as below

1. The screenshot does not have the GSAS-II tab: https://docs.mantidproject.org/nightly/interfaces/diffraction/Engineering%20Diffraction.html#parameters
https://docs.mantidproject.org/nightly/interfaces/diffraction/Engineering%20Diffraction.html#id1
https://docs.mantidproject.org/nightly/interfaces/diffraction/Engineering%20Diffraction.html#id2
2. The toolbar in the plot section does not contain the buttons up to date.
https://docs.mantidproject.org/nightly/interfaces/diffraction/Engineering%20Diffraction.html#id2

**Review by warunawickramasingha — APPROVED** [review]
Thanks for the changes and both the docs and dev-docs targets locally compiled successfully and looks up-to-date with the attached images. 
Happy to approve!

**Pull request merged**